tests/CanadaPost/Mocks/ncshipment-details-success.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<non-contract-shipment-details>
+    <final-shipping-point>J4W4T0</final-shipping-point>
+    <tracking-pin>11111118901234</tracking-pin>
+    <delivery-spec>
+        <service-code>DOM.EP</service-code>
+        <sender>
+            <company>Canada Post Corporation</company>
+            <contact-phone>555-555-5555</contact-phone>
+            <address-details>
+                <address-line-1>2701 Riverside Drive</address-line-1>
+                <city>Ottawa</city>
+                <prov-state>ON</prov-state>
+                <postal-zip-code>K1A0B1</postal-zip-code>
+            </address-details>
+        </sender>
+        <destination>
+            <name>John Doe</name>
+            <company>Consumer</company>
+            <address-details>
+                <address-line-1>2701 Receiver Drive</address-line-1>
+                <city>Ottawa</city>
+                <prov-state>ON</prov-state>
+                <country-code>CA</country-code>
+                <postal-zip-code>K1A0B1</postal-zip-code>
+            </address-details>
+        </destination>
+        <options>
+            <option>
+                <option-code>DC</option-code>
+            </option>
+        </options>
+        <parcel-characteristics>
+            <weight>15.000</weight>
+            <dimensions>
+                <length>1</length>
+                <width>1</width>
+                <height>1</height>
+            </dimensions>
+            <document>false</document>
+            <unpackaged>false</unpackaged>
+            <mailing-tube>false</mailing-tube>
+        </parcel-characteristics>
+        <preferences>
+            <show-packing-instructions>true</show-packing-instructions>
+            <show-postage-rate>false</show-postage-rate>
+            <show-insured-value>false</show-insured-value>
+        </preferences>
+    </delivery-spec>
+</non-contract-shipment-details>
```

Comment:
I think test data should be under a `fixtures` folder. `Mocks` refer to mock classes.